examples-standalone/dialogflow-integration/backend/webhook/README.md

```diff
@@ -0,0 +1,8 @@
+# Dialogflow: Webhook Template using Node.js and Cloud Functions for Firebase
+
+## Setup Instructions
+
+1. Go to the Dialogflow console and select *Fulfillment* from the left navigation menu.
+3. Enable *Inline Editor*, paste the code from `functions/index.js`, then click *Deploy*.
+4. Select *Intents* from the left navigation menu. Select the `Default Welcome Intent` intent, scroll down to the end of the page and click *Fulfillment*, check *Use webhook* and then click *Save*. This will allow you to have the welcome intent be a basic webhook intent to test.
```

Comment:
"Use webhook" option should be changed to "Enable webhook call for this intent". It is really minor and I'm not sure if it has been recently changed or I have misunderstood where to look :)